Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -238,7 +328,9 @@ internal extension URL {
 }
 
 extension HTTPClient {
+    /// Response execution context.
```

Comment:
Worth explaining that the user does not instantiate this themselves; the package does it.

Reply:
Is this better?

Reply:
It looks the same to me?

Reply:
Fixed, sorry, forgot to push.